server/osc_dash.py

```
@@ -763,6 +763,64 @@ async def api_upload_chunk(
+    """Atomically commit streamed temp file, count lines, and build index under lock in worker thread."""
+    lock = _get_target_lock(target_file.name)
+    with lock:
+        os.replace(tmp_target, target_file)
+        lines_count = _count_lines_fast(target_file)
+        windows_indexed = 0
+        try:
+            from backtest.index import build_index
+            _, total_snaps = build_index(target_file)
+            windows_indexed = total_snaps
+        except Exception:
+            pass
```

Ruff (0.16.3)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)